.circleci/config.yml

```diff
@@ -86,13 +55,16 @@ workflows:
             tags:
               only: /^v.*/
 
-      - architect/push-to-docker-legacy:
-          name: push-chart-operator-to-aliyun-legacy
-          image: "registry-intl.cn-shanghai.aliyuncs.com/giantswarm/chart-operator"
-          username_envar: "ALIYUN_USERNAME"
-          password_envar: "ALIYUN_PASSWORD"
+      - architect/push-to-app-catalog:
+          name: push-chart-operator-to-control-plane-app-catalog
+          app_catalog: "control-plane-catalog"
+          app_catalog_test: "control-plane-test-catalog"
+          chart: "chart-operator"
           requires:
-            - build
+            - push-chart-operator-to-quay
```

Comment:
We should install chart-operator in control planes from its catalog. So now its pushed to both control-plane-catalog and default-catalog.

Reply:
I'd like to add the app collection wiring later once we're ready to merge into master. As if the Helm 3 version of chart-operator gets deployed too early the migration logic will cause a mess.